helm/istio/charts/kiali/templates/kiali-cr.yaml

```diff
@@ -23,6 +23,9 @@ spec:
 {{- end }}
     image_name: "{{ .Values.hub }}/kiali"
     image_pull_policy: "{{ .Values.global.imagePullPolicy }}"
+{{- if .Values.global.imagePullSecrets }}
+    image_pull_secrets: "{{ .Values.global.imagePullSecrets }}"
```

Comment:
here you set the value of the image_pull_secrets, but then you have a `{{ - range}}` on the next line with the same .Values.global.imagePullSecrets. That doesn't seem right - it looks like you are putting the secrets list in here twice?